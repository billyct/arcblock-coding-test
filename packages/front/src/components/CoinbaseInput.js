const CoinbaseInput = () => (
  <div
    className='py-2'
  >
    <div className='uppercase font-bold bg-green-100 inline-block px-4 py-2'>
      Coinbase
    </div>

  </div>
)

export default CoinbaseInput
