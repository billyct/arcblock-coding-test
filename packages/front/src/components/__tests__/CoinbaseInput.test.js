import {render} from '@testing-library/react'

import CoinbaseInput from '../CoinbaseInput'

test('CoinbaseInput Component', () => {
  const {container} = render(<CoinbaseInput/>)
  expect(container).toMatchSnapshot()
})
