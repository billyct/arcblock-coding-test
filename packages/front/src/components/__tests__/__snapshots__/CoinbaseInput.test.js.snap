// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CoinbaseInput Component 1`] = `
<div>
  <div
    class="py-2"
  >
    <div
      class="uppercase font-bold bg-green-100 inline-block px-4 py-2"
    >
      Coinbase
    </div>
  </div>
</div>
`;
